app/jobs/create_tap_activity.rb

```diff
@@ -0,0 +1,20 @@
+require 'notifications/client'
+
+class CreateTapActivity < ApplicationJob
+  class UnableToCreateSummaryDocumentActivity < StandardError; end
+
+  queue_as :default
+
+  def perform(appointment_summary, guider)
+    telephone_appointment = TelephoneAppointments::SummaryDocumentActivity.new(
+      appointment_id: appointment_summary.reference_number,
+      owner_uid: guider.uid,
+      delivery_method: appointment_summary.requested_digital ? 'digital' : 'postal'
```

Comment:
I can't help feeling this API should have also accepted `requested_digital: true|false`... and done any conversion internally.